packages/nimble-components/docs/component-status.stories.ts

```diff
@@ -23,7 +23,7 @@ const components = [
     },
     {
         componentName: 'Anchor',
-        componentHref: '/?path=/docs/components-anchor--docs',
+        componentHref: '../?path=/docs/components-anchor--docs',
```

Comment:
We just want to change the search query parameter on the url right? Can we just do that?:

```suggestion
        componentHref: '?path=/docs/components-anchor--docs',
```

Hopefully that works because what is being shown doesn't make sense:

![image](https://github.com/ni/nimble/assets/1588923/7721d5f7-e26d-4c6c-84de-2e570c69ca4c)

Notice how `storybook` gets removed from the path.

Compared to:

![image](https://github.com/ni/nimble/assets/1588923/1f9dc302-3aa7-475d-9706-e633706bb4bf)

You can also test this stuff out by doing a full build (or build of storybook and packages/site) and then `npm run start:eleventy -w packages/site`

Reply:
If we end up needing to hold onto this weird path thing can you add a comment on the comments var linking to the issue. In particular because the relative path doesn't make sense from what I can see so there must be something else happening...

Reply:
Ohhh I see, the component is inside an iframe so the page url actually looks like `https://nimble.ni.dev/storybook/iframe.html?viewMode=docs&id=component-status--docs`

So we want the url to be resolved from the current directory like:
```suggestion
        componentHref: './?path=/docs/components-anchor--docs',
```

Since the iframe.html is "in the same directory" as the root index page. BUT we don't want to navigate the iframe context we want to navigate the top level browser. So the anchor column should be configured `<nimble-table-column-anchor target="_top">`

Reply:
Thanks for the guidance! I confirmed that your suggested approach works for both the local site build and a local storybook build. And that the previously suggested approach within markdown works for those too (I think Storybook has some special processing for that case). 

So I updated the status table to use the new approach and updated our docs with guidance for markdown links and anchor links.